production/helm/templates/loki/_helpers.tpl

```diff
@@ -41,3 +41,14 @@ Create the name of the service account
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return the appropriate apiVersion for networkpolicy.
+*/}}
+{{- define "loki.networkPolicy.apiVersion" -}}
+{{- if semverCompare ">=1.4-0, <1.7-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
I don't think this function is necessary, just use `networking.k8s.io/v1`. Helm charts in stable have a [policy](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#compatibility) to only support the current and previous Kubernetes versions and to use the latest apiVersion for new features.

Reply:
Ah, and 1.10 is required anyway. Thanks; I'll modify that.